gateway/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM eclipse-temurin:21-jdk-alpine
+
+ARG JAR_FILE=target/*.jar
+
+COPY ${JAR_FILE} gateway.jar
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)